pycbc/events/ranking.py

```diff
@@ -57,7 +57,7 @@ def newsnr_sgveto_psdvar(snr, bchisq, sgchisq, psd_var_val):
     variation statistic """
     nsnr = numpy.array(newsnr_sgveto(snr, bchisq, sgchisq), ndmin=1)
     psd_var_val = numpy.array(psd_var_val, ndmin=1)
-    lgc = psd_var_val >= 1.8
+    lgc = psd_var_val >= 1.2
```

Comment:
maybe want to add some comment about why 1.2 was chosen?

Reply:
1.2 is the maximum value obtained after testing the code over gaussian noise. Are you suggesting me to write this down as a comment?

Reply:
Yes, since it's a hard-coded number and not clear to someone reading the code why this choice was made.